weave/scorers/scorer_types.py

```diff
@@ -92,10 +92,10 @@ def validate_device(cls, v: Union[str, "torch.device"]) -> "torch.device":
             return set_device(v)
 
     def model_post_init(self, __context: Any) -> None:
-        if self._pipeline is None:
-            self.load_pipeline()
+        if not hasattr(self, "_pipeline"):
```

Comment:
I think the old one makes more sense.  I would do:
```py
if self._pipeline is None:
    self._pipeline = self.load_pipeline()
```

Reply:
updated